lib/driverProviders/sauce.ts

```diff
@@ -63,10 +63,11 @@ export class Sauce extends DriverProvider {
     this.config_.capabilities['username'] = this.config_.sauceUser;
     this.config_.capabilities['accessKey'] = this.config_.sauceKey;
     this.config_.capabilities['build'] = this.config_.sauceBuild;
-    let auth = 'https://' + this.config_.sauceUser + ':' + this.config_.sauceKey + '@';
-    this.config_.seleniumAddress = auth +
-        (this.config_.sauceSeleniumAddress ? this.config_.sauceSeleniumAddress :
-                                             'ondemand.saucelabs.com:443/wd/hub');
+
+    let auth = this.config_.sauceUser + ':' + this.config_.sauceKey + '@';
+    this.config_.seleniumAddress = this.config_.sauceSeleniumAddress ?
+        'http://' + auth + this.config_.sauceSeleniumAddress :
```

Comment:
@seanmay This would not be secure (http, not https) when relaying Selenium commands through Sauce Connect Tunnel. (i.e. when pointing at Sauce connect's selenium listener/relay, rather than sending selenium command over the internet to Sauce's https endpoint).